meta-resin-common/recipes-containers/docker-disk/files/entry.sh

```diff
@@ -10,7 +10,7 @@ PARTITION_SIZE=${PARTITION_SIZE:=1024}
 
 # Create a BTRFS disk image
 dd if=/dev/zero of=/export/data_disk.img bs=1M count=$PARTITION_SIZE
-mkfs.btrfs /export/data_disk.img
+mkfs.btrfs --mixed --metadata=single /export/data_disk.img
```

Comment:
@petrosagg we have had btrfs corruption happen quite a few times, possible due to SD card corruption, so it's not _unreasonable _to have duplication. Perhaps in the past some cases have been recoverable due to this duplication. However, it's a hard one to repro and probably having more metadata space available matters more so it's not a huge big deal.

On mixed mode, I am not saying we shouldn't do it, let's just see behaviour in low space to see what happens in practice. According to the quote you've given they wouldn't recommend mixed mode for any resin device I've seen (they're all >1GiB) and I've seen out of space on >5GiB file systems so I wouldn't take the recommendation to imply that the 'unable to delete' scenario will be avoided.

So all I propose is that we recreate low space in a filesystem with mixed mode turned on, see how it behaves compared to current. If it totally eliminates the 'cannot delete' scenario we're laughing, if it acts as now then it's still fine to carry on since it won't be any worse + provide more metadata space but I want to see if any _new_ kind of weirdness hits with this.

It doesn't necessarily have to block the PR, as long as we do it at some point (I don't think it'll take long!)